### PR TITLE
Fixed: Added wiki fragments to indexer health checks

### LINKED ISSUE #6216: Custom script variable HealthCheck Wiki returns wrong link
**Describe the bug**
When calling `radarr_health_issue_wiki` from a custom script, it returns wrong wiki link from Servarr Wiki for `IndexerRssCheck` and `IndexerSearchCheck`.

**To Reproduce**
Create a custom script and call `radarr_health_issue_wiki`.

**Expected behavior**
It should return the right link.

**Platform Information:**
 - OS: Windows 10 20H2 build 19042.906
 - Docker:No
 - Mono or .NET Version: 5.0.5
 - Radarr Version: 3.1.0.4893
 - Radarr Branch: develop

### PR DESCRIPTION
**Pull request by RobinDadswell**
#### Database Migration
NO

#### Description
Added some specific Wiki Fragments for indexers

#### Todos
- [ ] Tests
- [ ] Wiki Updates

#### Issues Fixed or Closed by this PR

* Fixes #6216

### CODE REVIEW

**Comment by RobinDadswell** [comment]
@Qstick / @ta264  we have an issue at the moment with Wiki Links where if there isn't a fragment specifically stated we use the message, which is a localised string - any thoughts on us changing that behaviour at the moment? 
The line in question is https://github.com/Radarr/Radarr/blob/bb721272ca1ef4336fcc159563a33794345a185b/src/NzbDrone.Core/HealthCheck/HealthCheck.cs#L32

**Comment by Qstick** [comment]
> @Qstick / @ta264 we have an issue at the moment with Wiki Links where if there isn't a fragment specifically stated we use the message, which is a localised string - any thoughts on us changing that behaviour at the moment?
> The line in question is
> 
> https://github.com/Radarr/Radarr/blob/bb721272ca1ef4336fcc159563a33794345a185b/src/NzbDrone.Core/HealthCheck/HealthCheck.cs#L32

Yea, we should likely add them all explicitly or use the `en` translation for the fragment.

**Review by Qstick — APPROVED** [review]

File: src/NzbDrone.Core/HealthCheck/Checks/IndexerSearchCheck.cs [hunk 1]
```diff
@@ -25,21 +25,21 @@ public override HealthCheck Check()
...
-                return new HealthCheck(GetType(), HealthCheckResult.Warning, _localizationService.GetLocalizedString("IndexerSearchCheckNoInteractiveMessage"));
+                return new HealthCheck(GetType(), HealthCheckResult.Warning, _localizationService.GetLocalizedString("IndexerSearchCheckNoInteractiveMessage"), "#no_indexers_available_with_interactive_search_enabled");
             }
 
             var active = _indexerFactory.AutomaticSearchEnabled(true);
 
             if (active.Empty())
             {
-                return new HealthCheck(GetType(), HealthCheckResult.Warning, _localizationService.GetLocalizedString("IndexerSearchCheckNoAvailableIndexersMessage"));
+                return new HealthCheck(GetType(), HealthCheckResult.Warning, _localizationService.GetLocalizedString("IndexerSearchCheckNoAvailableIndexersMessage"), "#indexers_are_unavailable_due_to_failures");
```
Comment (Qstick) [inline src/NzbDrone.Core/HealthCheck/Checks/IndexerSearchCheck.cs:42]: Should this be a different fragment than rss, or do we want both going to same wiki?

**Review by Qstick — COMMENTED** [review]
One comment, I’m good either way on. Otherwise looks solid

File: src/NzbDrone.Core/HealthCheck/Checks/IndexerSearchCheck.cs (hunk 1 above)
Comment (RobinDadswell) [reply src/NzbDrone.Core/HealthCheck/Checks/IndexerSearchCheck.cs:42]: https://wiki.servarr.com/System_status#indexers_are_unavailable_due_to_failures - the wiki link on that one is probably relevant for both so up to you on that one :)

**Review by Qstick — APPROVED** [review]

**Review by Qstick — APPROVED** [review]

**Pull request merged**